devfile/devworkspace-operator
# feat: Add DWO icon for OperatorHub

### What does this PR do?
Adds an operator icon for DWO (when being installed from OLM or from OperatorHub) 

### What issues does this PR fix or reference?
Fix #892

### Is it tested? How?
To manually test this:
1. Start up an OpenShift cluster and log in to it
2. Set the `DWO_BUNDLE_IMG` and `DWO_INDEX_IMG` environment variables to point to your own quay.io repositories
3. To get things to work, I also had to remove the `updateStrategy` field from `catalog-source.yaml`
```diff
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: devworkspace-operator-catalog
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: quay.io/devfile/devworkspace-operator-index:next
  publisher: Red Hat
  displayName: DevWorkspace Operator Catalog
- updateStrategy:
-   registryPoll:
-     interval: 1d
```
4. run `make generate_olm_bundle_yaml build_bundle_and_index register_catalogsource`
5. Open up the OpenShift console, go to the OperatorHub page, search for devworkspace-operator and ensure the icon is present.
![image](https://user-images.githubusercontent.com/10300119/184926027-fd5741f0-7eee-40be-ba6c-628484873075.png)


### PR Checklist
